CMakeLists.txt

```diff
@@ -396,11 +396,11 @@ endif ()
 # ----------------------------------------------------------------------------
 # installation directories
 if (OS_WINDOWS)
-  set (RUNTIME_INSTALL_DIR Bin)
-  set (LIBRARY_INSTALL_DIR Lib)
-  set (INCLUDE_INSTALL_DIR Include)
-  set (CONFIG_INSTALL_DIR  CMake)
-  set (PKGCONFIG_INSTALL_DIR)
+  set (RUNTIME_INSTALL_DIR "bin")
```

Comment:
Required only changes for `CONFIG_INSTALL_DIR` (and `install` command), because vcpkg allow only use this folder ATM. Other changes just for consistency with this change and other CMake projects.